qadence/backends/pulser/cloud.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import os
+from functools import lru_cache
+from typing import Optional
+
+from pasqal_cloud import AUTH0_CONFIG, PASQAL_ENDPOINTS, SDK, Auth0Conf, Endpoints, TokenProvider
+
+DEFAULT_CLOUD_ENV = "prod"
```

Comment:
Can you make a `StrEnum` with the available environments instead of using a simple string?

Reply:
Sure